cms/static/js/certificates/models/signatory.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'backbone', 'underscore', 'underscore.string', 'gettext', 'backbone-relational'
+],
+function(Backbone, _, str, gettext) {
+    'use strict';
+    console.log('certificate_relational_model.Signatory.start');
+    _.str = str;
+
+    var Signatory = Backbone.RelationalModel.extend({
+        idAttribute: "id",
+        urlRoot: '/api/',
+        defaults: {
+            name: 'Instructor name',
```

Comment:
Instead of 'Instructor' we should say 'Signatory' name here -- it's my understanding that not all signatories will be instructors for the course.